packages/vite/src/vite.ts

```diff
@@ -89,7 +89,6 @@ export const bundle: NuxtBuilder['bundle'] = async (nuxt) => {
         },
         resolve: {
           alias: {
-            [basename(nuxt.options.dir.assets)]: resolve(nuxt.options.srcDir, nuxt.options.dir.assets),
```

Comment:
I think we need to keep this entry here and in the webpack presets until https://github.com/vuejs/core/pull/13462 merges.

see https://github.com/nuxt/nuxt/pull/32119/commits/03d7c6e4fd14df072c4f5945e0ede7f399f58367